### PR TITLE
feat: updating alerts to use Paragon Alert over custom (AA-914)

### PR DESCRIPTION
**Pull request by ciduarte**

### CODE REVIEW

File: src/courseware/course/course-exit/CourseCelebration.jsx [hunk 1]
```diff
@@ -285,34 +286,37 @@ function CourseCelebration({ intl }) {
...
-                    variant={buttonVariant}
-                    href={buttonLocation}
-                    onClick={() => logClick(org, courseId, administrator, buttonEvent)}
-                  >
-                    {buttonText}
-                  </Button>
-                )}
-                {buttonSuffix}
+          <Alert variant="success" icon={CheckCircle}>
+            <div className="row w-100 m-0">
```
Comment (ciduarte) [inline src/courseware/course/course-exit/CourseCelebration.jsx:290]: These were the only lines that really changed. The added div on line 290 caused more indents

File: src/shared/streak-celebration/StreakCelebrationModal.jsx [hunk 2]
```diff
@@ -134,18 +134,20 @@ function StreakModal({
...
-              &nbsp;{intl.formatMessage(messages.streakDiscountMessage)}&nbsp;
-              <FormattedMessage
-                id="learning.streakCelebration.streakAA759EndDateMessage"
-                defaultMessage="Ends {date}."
-                values={{
-                  date: new Date('2021-7-20 00:00').toLocaleDateString({ timeZone: 'UTC' }),
-                }}
-              />
+          <Alert variant="success" className="px-0">
+            <div className="d-flex">
```
Comment (ciduarte) [inline src/shared/streak-celebration/StreakCelebrationModal.jsx:138]: Popped in this div, caused indent changes

File: src/generic/user-messages/Alert.jsx [hunk 3]
```diff
@@ -1,82 +1,47 @@
...
-              onClick={onDismiss}
-              alt={intl.formatMessage(messages.close)}
-              variant="primary"
-            />
-          </div>
-        )}
-      </div>
-      {footer}
-    </div>
+    <ParagonAlert
```
Comment (ciduarte) [inline src/generic/user-messages/Alert.jsx:36]: I don't love the fact that we're wrapping the Paragon component in a custom one, but courseware still uses this custom Alert framework and I wanted to update it without having to refactor a ton.

File: src/courseware/course/course-exit/ProgramCompletion.jsx [hunk 4]
```diff
@@ -41,54 +41,56 @@ function ProgramCompletion({
...
-          <FormattedMessage
-            id="courseExit.programCompletion.dashboardMessage"
-            defaultMessage="To view your certificate status, check the Programs section of your {programLink}."
-            values={{ programLink }}
-          />
-        </p>
-        {type === 'microbachelors' && (
-          <>
+    <Alert variant="primary" className="my-3" data-testid="program-completion">
+      <div className="d-flex">
```
Comment (ciduarte) [inline src/courseware/course/course-exit/ProgramCompletion.jsx:45]: Just updated these two lines, the rest are indentation changes

File: src/course-home/outline-tab/alerts/private-course-alert/messages.js [hunk 5]
```diff
@@ -1,10 +1,11 @@
 import { defineMessages } from '@edx/frontend-platform/i18n';
 
 const messages = defineMessages({
-  enroll: {
+  toAccess: {
     id: 'alert.enroll',
-    defaultMessage: 'You must be enrolled in the course to see course content.',
-    description: 'Text instructing the learner to enroll in the course in order to see course content.',
+    defaultMessage: ' to access the full course.',
```
Comment (Dillon-Dumesnil) [inline src/course-home/outline-tab/alerts/private-course-alert/messages.js:6]: Weird, but okay

**Review by mikix — APPROVED** [review]
Sure? Looks OK - love standardizing on paragon code.

File: src/generic/user-messages/Alert.jsx [hunk 6]
```diff
@@ -1,82 +1,47 @@
...
-import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { Alert as ParagonAlert } from '@edx/paragon';
+import { CheckCircle, Info, WarningFilled } from '@edx/paragon/icons';
 
 import { ALERT_TYPES } from './UserMessagesProvider';
-import './Alert.scss';
-import messages from '../messages';
 
-function getAlertClass(type) {
+function getAlertVariant(type) {
```
Comment (Dillon-Dumesnil) [inline src/generic/user-messages/Alert.jsx:8]: `switch`?

**Review by Dillon-Dumesnil — APPROVED** [review]
Sure

**Pull request merged**